src/index.json

```diff
@@ -285,7 +285,7 @@
         "webapp": [
             {
                 "filename": "webapp-0.2.9-py2.py3-none-any.whl",
-                "sha256Digest": "f75b3e09cd3d99a0aaf47d9cf1f3792f6016bbbab4815d42afe6c55e5e306e1f",
+                "sha256Digest": "c6f9957a8326b2afde645a7538e4c5981145390d133cfa08a7a9b410ee135569",
```

Comment:
I think the CI may be failing because you didn't update the azext_metadata.json. 
index.json's metadata should reflect [setup.py](https://github.com/Azure/azure-cli-extensions/blob/master/src/webapp/setup.py) which pulls some information from [azext_metada.json](https://github.com/Azure/azure-cli-extensions/blob/master/src/webapp/azext_webapp/azext_metadata.json).